### PR TITLE
Porfolio Website with Media Quary

### PR DESCRIPTION
**Pull request by XRichardroidX**
I have made a responsive project with the help of media query and it works well on mobile and desktop view.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -203,54 +297,56 @@ <h4 class="technology-text">Skills</h4>
...
+                    </li>
+                    <li>
+                        <input type="email" name="email" id="email" placeholder="Email address" required/>
+                    </li>
+                    <li>
+                        <textarea type="textarea" name="textarea" id="textarea" placeholder="Enter text here" maxlength="500" required></textarea>
+                    </li>
+                </ul>
+                         <button class="btn" type="submit">Get in touch</button>
+                    <img id="last-image" src="images/last-image.svg" alt="last-image">
```
Comment (mgmediaweb) [inline index.html:317]: Kindly, I suggest you use background images for these designs

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @KIsmoilov, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost1

## Highlights 💯 
- Professional-looking readme file. ✔️ 
- Clean and tidy code. ✔️ 
- Good job following the design. ✔️ 

## Required Changes ♻️
- Kindly, I suggest you create a more descriptive PR, detailing what you are presenting to the reviewer, this will help us better understand your work.
- Kindly, I suggest you check why a horizontal scroll appears on your page, check what element is producing it and adjust it.
image

- Kindly, I suggest you review the original Figma layout, it should display these cards in full screen (with margins)
image

- Kindly, I suggest you to put the proper font, size and color follow the Figma design
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly, I suggest you indicate more descriptive branch names

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -203,54 +297,56 @@ <h4 class="technology-text">Skills</h4>
...
-        <div class="contact-form">
-            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-        </div>
-        <form action="https://formspree.io/f/xrgdqvko" method="post">
-            <ul class="form">
-                <li>
-                    <input type="text" name="name" id="name" placeholder="Full name" maxlength="30" required/>
+    <section class="contact-section">
+            <div class="contact-form">
+                <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
```
Comment (awais-amjed) [inline index.html:302]: As mentioned by the previous reviewer, kindly change the font-family of this text to `Inter` 🤗

Your design:

image

Figma design:

image

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @XRichardroidX , 

Great work on making the changes requested by a previous reviewer 👏👏👏
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @XRichardroidX 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- No Linter errors ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**